### PR TITLE
IGAPP-960: Deliver hidden files and apple-app-site-association

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: build-configs/BuildConfigType.ts [hunk 1]
```diff
@@ -10,14 +10,16 @@ import { ThemeType } from './ThemeType'
 // If you change this make sure you are not navigating to the landing screen upon closing the intro slides.
```
Comment (klinzo) [inline build-configs/BuildConfigType.ts:10]: Mostly just moved comments to the right place

**Comment by klinzo** [comment]
Testable here test.integreat.app/apple-app-site-association and here https://test.integreat.app/.well-known/assetlinks.json

**Review by SteffiStuffel — APPROVED** [review]
Do not really get what you have done here in detail, but looked good.

**Comment by klinzo** [comment]
> Do not really get what you have done here in detail, but looked good.

@SteffiStuffel two things:
1. With our previous delivery method using sftp hidden files and directories (i.e. those starting with a dot `.`) where not delivered/served. Therefore e.g. `.well-known/assetlinks.json` was not available and therefore we got the errors in google ads that our deep links are not correctly configured (see https://issues.tuerantuer.org/browse/IGAPP-857).
2. Previously apple-app-site-association was not part of our repo but hardcoded on the server side. This was imo bad for documentation/responsibility purposes and was also broken because of some changes there. So now the apple-app-site-association is also created and delivered from this repo together with the rest of the delivery.

**Pull request merged**